Extensions/servant.js

```diff
@@ -1141,11 +1141,11 @@ XKit.extensions.servant = new Object({
 				}
 
 				if (m_post !== "") {
-					var to_run = function() {
+					var to_run = function(post) {
 						eval(parameter_fixed + "\n//# sourceURL=xkit/servant/servant" + (new Date()).getTime() + ".js");
 					};
 
-					to_run();
+					to_run(m_post);
```

Comment:
the examples seem to imply that the post is passed as an HTML element rather than a jQuery object, which is what `m_post` is; out of consideration for vanilla JS users we should actually do this
```js
to_run(m_post[0]);
```